### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by Paulao17**
Translated strings into French. Did not modify `credits_content_translator`.
Thank you !

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+	<!-- Loading -->
+	<string name="loading_title">Test du support…</string>
+	<string name="loading_body">Veuillez patienter tandis que je test le téléchargement du support…</string>
+
+	<!-- Credits -->
+	<string name="credits_version">v%1$s</string>
+	<string name="title_activity_credits">A propos de &appname;</string>
+	<string name="logo_description">Logo</string>
+	<string name="credits_creator">par <a href="https://michael.standen.link">&author;</a></string>
+	<string name="credits_content1">Phone Saver est une simple application pour sauvegarder
```
Comment (ScreamingHawk) [inline app/src/main/res/values-fr/strings.xml]: Could you please use `&appname;` here instead of `Phone Saver`

Comment (Paulao17) [reply app/src/main/res/values-fr/strings.xml]: Will do

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+		des fichiers sur votre téléphone via la fonctionnalité partager.
+	</string>
+	<string name="credits_content2">Vous pouvez accéder au code source, faire connaitre des bugs, et demander des fonctionnalités sur
+		<a href="https://github.com/ScreamingHawk/phone-saver">Github</a>.
+	</string>
+	<string name="credits_content3">Cette application est complètement open source; elle a pour license la
+		<a href="https://tldrlegal.com/license/mit-license">license MIT</a>.
+	</string>
+	<string name="credits_content_translator">Cette traduction de &appname; a été proposée par
+		<a href="https://michael.standen.link">&author;</a>.
```
Comment (ScreamingHawk) [inline app/src/main/res/values-fr/strings.xml]: You should absolutely change this to yourself!!! It's there to credit the kind souls like yourself who take the time to contribute to open source :)

**Comment by ScreamingHawk** [comment]
Thank you so much! This will be available in the next release

**Pull request merged**